runner.sh

```diff
@@ -50,7 +50,7 @@ doChecks=true
 doRun=true
 autofix=false
 failfast=false
-pattern="-name \"*\""
+pattern="-and -name '*' -and ! -wholename '*federated_learning*'"
```

Comment:
Hi @rijobro I exclude the `federated_learning` folder by default, because it will likely to use multiple node, which is not feasible for the CI/CD at the moment.